manifests/params.pp

```diff
@@ -90,6 +94,9 @@
         } else {
           $repo_opt = undef
         }
+      } elsif (versioncmp($::operatingsystemmajrelease, '6') == 0 and $::operatingsystem == 'OracleLinux') {
```

Comment:
It might be better to write this test similar to the ones in L63 and L73:

```
if versioncmp($::operatingsystemrelease, '7.0') < 0 ...
```

assuming, there is no OL5. Or at least nobody is trying to run docker on it.